app/views/spots/index.html.erb

```diff
@@ -5,7 +5,7 @@
   <tbody>
     <% @spots.each do |spot| %>
       <tr>
-        <td> <%= link_to spot.name, spots_path(spot) %> </td>
+        <td> <%= link_to spot.name, spot_path(spot.id) %> </td>
```

Comment:
I believe you could do: `link_to spot.name, spot`, but I might be wrong about that.